MobileWallet/Screens/Home/TransactionHistory/TransactionTableViewModel.swift

```diff
@@ -177,6 +180,11 @@ class TransactionTableViewModel: NSObject {
         gifID = giphyDd
 
         super.init()
+
+        Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { [weak self] (_) in
```

Comment:
What about having just one timer in the TableVC that just triggers `tableView.reloadData()`? Would that not work the same?

Reply:
We do not need to reload table view, only label text. Also we can improve timer and update first hour with 1min interval and later 1h .. 1d..etc

Reply:
Also when we use this way we are updating only visible cells

Reply:
And wouldn't reloading the table restart all the GIFs?

Reply:
@kukabi  great, one more reason to use this way)

Reply:
But I think you should keep the timer reference and invalidate in `deinit` or something.

Reply:
Ok cool. Yeah does it keep the reference this way or do you have to kill it manually?

Reply:
Ok just tested on a playground, it kills the timer once the objects gets `deinit`ed.

Reply:
You can do 60 seconds though, every second might be a bit too much. So it will display `Now` until 1 minute, then update every minute.